lib/v2/datasource/remote/model/moon_phase_model.dart

```diff
@@ -0,0 +1,36 @@
+class MoonPhasesList {
```

Comment:
I dont think this class is needed, Its just a class to hold a list.

You can just return a list of MoonPhase without needing this class

Reply:
You can use this in the  "ProposalsRepository" - getMoonPhases

```
return http
        .post(proposalsURL, headers: headers, body: request)
        .then((http.Response response) => mapHttpResponse(response, (dynamic body) {
              if (body['rows'] != null) {
                return [];
              } else {
                return body['rows'].map<MoonPhase>((i) => MoonPhase.fromJson(i)).toList();
              }
            }))
        .catchError((error) => mapHttpError(error));
```

Reply:
I did it in that way precisely to avoid to write that logic code in the call.

I thought the standard was to return only one model.

```
        .then((http.Response response) => mapHttpResponse(response, (dynamic body) {
              return SomeModel.fromJson(body);
            }))
        .catchError((error) => mapHttpError(error));
```

Reply:
ideally not. 
I think we can avoid the null check, and just return the map 

return http
        .post(proposalsURL, headers: headers, body: request)
        .then((http.Response response) => mapHttpResponse(response, (dynamic body) {

                return body['rows'].map<MoonPhase>((i) => MoonPhase.fromJson(i)).toList();

            }))
        .catchError((error) => mapHttpError(error));